### PR TITLE
[onert] Support ShapeInference on BCQFullyConnected

### PR DESCRIPTION
**Pull request by dayo09**
- support shape inference on BCQFullyConnected

ONE-DCO-1.0-Signed-off-by: dayo09 <dayg502@gmail.com>

### CODE REVIEW

File: runtime/onert/core/src/compiler/StaticShapeInference.cc [hunk 1]
```diff
@@ -182,6 +182,34 @@ void StaticShapeInferer::visit(const ir::operation::BatchMatMul &op)
...
+  const auto &input = _operands.at(input_idx);
+
+  const auto cluster_idx{
+      op.getInputs().at(ir::operation::BCQFullyConnected::Input::WEIGHTS_CLUSTERS)};
+  const auto &cluster = _operands.at(cluster_idx);
+
+  const auto output_idx = op.getOutputs().at(0);
+  ir::Operand &output = _operands.at(output_idx);
+
+  if (!cluster.isConstant())
```
Comment (llFreetimell) [inline runtime/onert/core/src/compiler/StaticShapeInference.cc:197]: AFAIK, cluster is always constant.
Is this condition correct...?

Comment (dayo09) [reply runtime/onert/core/src/compiler/StaticShapeInference.cc]: I have changed.

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**